packages/diagram_capture/test/diagram_capture_test.dart

```diff
@@ -158,22 +160,35 @@ void main() {
       final DiagramController controller = new DiagramController(
         builder: (BuildContext context) => new TestAnimatedDiagram(key: key, size: 1.0),
         outputDirectory: outputDir,
-        frameFilenameGenerator: filenameGenerator,
+        filenameGenerator: filenameGenerator,
         pixelRatio: 1.0,
         screenDimensions: const Size(100.0, 100.0),
       );
 
       controller.builder = (BuildContext context) => new TestAnimatedDiagram(key: key, size: 50.0);
-      final List<File> outputFiles = await controller.drawAnimatedDiagramToFiles(
+      final File outputFile = await controller.drawAnimatedDiagramToFiles(
         end: const Duration(milliseconds: 1200),
         frameDuration: const Duration(milliseconds: 200),
```

Comment:
can we do 60Hz animations instead of 5Hz animations?

Reply:
oh never mind this is a test

Reply:
Yes, it's just a test.  Also, yes, we can do 60hz animations. I just didn't want to produce too much data for the test.